Unexpected behavior when OPENMDAO_REPORTS=0

### Description

I noticed `test_ex_two_burn_orbit_raise_bokeh_plots.py` was failing when I had the env var `OPENMDAO_REPORTS=0`.

Interestingly the similar matplotlib version of the test (nor any other test in the full test suite) is sensitive to this variable (see below).

The test should probably have a  `with set_env_vars_context(OPENMDAO_REPORTS=1):`, but on the surface, it's interesting that the other test(s) don't have the same behavior...

### Example

```
[16:06:39](dymos) swryan@Chimaera:~/dev/dymos/dymos/examples/finite_burn_orbit_raise/test$ testflo -v test_ex_two_burn_orbit_raise_bokeh_plots.py
test_ex_two_burn_orbit_raise_bokeh_plots.py:TestExampleTwoBurnOrbitRaise.test_mpl_plots ... OK (00:00:14.94, 404 MB)
test_ex_two_burn_orbit_raise_bokeh_plots.py:TestExampleTwoBurnOrbitRaise.test_bokeh_plots ... OK (00:00:15.84, 395 MB)


OK

Passed:  2
Failed:  0
Skipped: 0


Ran 2 tests using 12 processes
Wall clock time:   00:00:24.64

[16:07:34](dymos) swryan@Chimaera:~/dev/dymos/dymos/examples/finite_burn_orbit_raise/test$ export OPENMDAO_REPORTS=0

[16:07:47](dymos) swryan@Chimaera:~/dev/dymos/dymos/examples/finite_burn_orbit_raise/test$ testflo -v test_ex_two_burn_orbit_raise_bokeh_plots.py
test_ex_two_burn_orbit_raise_bokeh_plots.py:TestExampleTwoBurnOrbitRaise.test_bokeh_plots  ... FAIL (00:00:13.49, 397 MB)
/home/swryan/miniconda3/envs/dymos/lib/python3.12/site-packages/openmdao/core/group.py:1101: DerivativesWarning:Constraints or objectives [ode_eval.control_interp.control_rates:u1_rate2] cannot be impacted by the design variables of the problem because no partials were defined for them in their parent component(s).
/home/swryan/miniconda3/envs/dymos/lib/python3.12/site-packages/openmdao/core/group.py:1101: DerivativesWarning:Constraints or objectives [ode_eval.control_interp.control_rates:u1_rate2] cannot be impacted by the design variables of the problem because no partials were defined for them in their parent component(s).

Traceback (most recent call last):
  File "/home/swryan/dev/dymos/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py", line 51, in test_bokeh_plots
    self.assertTrue(html_file.exists(), msg=f'{html_file} does not exist!')
AssertionError: False is not true : reports/testflo/traj_results_report.html does not exist!

test_ex_two_burn_orbit_raise_bokeh_plots.py:TestExampleTwoBurnOrbitRaise.test_mpl_plots ... OK (00:00:13.91, 401 MB)


The following tests failed:
test_ex_two_burn_orbit_raise_bokeh_plots.py:TestExampleTwoBurnOrbitRaise.test_bokeh_plots


Passed:  1
Failed:  1
Skipped: 0


Ran 2 tests using 12 processes
Wall clock time:   00:00:21.15

(dymos) swryan@Chimaera:~/dev/dymos/dymos/examples/finite_burn_orbit_raise/test$
```

### Dymos Version

1.11.0

### Relevant environment information

_No response_

--- a/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
+++ b/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
@@ -44,8 +44,8 @@ class TestExampleTwoBurnOrbitRaise(unittest.TestCase):
         with set_env_vars_context(OPENMDAO_REPORTS='1'):
             with dm.options.temporary(plots='bokeh'):
                 p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
-                                                compressed=False, optimizer='SLSQP', show_output=False,
-                                                run_driver=False)
+                                                 compressed=False, optimizer='SLSQP', show_output=False,
+                                                 run_driver=False)
 
                 html_file = pathlib.Path(_get_reports_dir(p)) / 'traj_results_report.html'
                 self.assertTrue(html_file.exists(), msg=f'{html_file} does not exist!')
@@ -54,12 +54,12 @@ class TestExampleTwoBurnOrbitRaise(unittest.TestCase):
     def test_mpl_plots(self):
         with dm.options.temporary(plots='matplotlib'):
             p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
-                                            compressed=False, optimizer='SLSQP', show_output=False,
-                                            run_driver=False)
+                                             compressed=False, optimizer='SLSQP', show_output=False,
+                                             run_driver=False)
 
             expected_files = ('deltav.png', 'r.png', 'accel.png',
-                            'u1.png', 'vr.png', 'pos_x.png',
-                            'vt.png', 'pos_y.png', 'theta.png')
+                              'u1.png', 'vr.png', 'pos_x.png',
+                              'vt.png', 'pos_y.png', 'theta.png')
 
             for file in expected_files:
                 plotfile = pathlib.Path(_get_reports_dir(p)).joinpath('plots') / file

--- a/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
+++ b/dymos/examples/finite_burn_orbit_raise/test/test_ex_two_burn_orbit_raise_bokeh_plots.py
@@ -11,7 +11,7 @@ except ImportError:
 
 from openmdao.utils.testing_utils import require_pyoptsparse
 from openmdao.utils.general_utils import set_pyoptsparse_opt
-from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.utils.testing_utils import use_tempdirs, set_env_vars_context
 
 
 import dymos as dm
@@ -41,30 +41,29 @@ class TestExampleTwoBurnOrbitRaise(unittest.TestCase):
 
     @unittest.skipIf(not bokeh_available, 'bokeh unavailable')
     def test_bokeh_plots(self):
-        dm.options['plots'] = 'bokeh'
+        with set_env_vars_context(OPENMDAO_REPORTS='1'):
+            with dm.options.temporary(plots='bokeh'):
+                p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
+                                                compressed=False, optimizer='SLSQP', show_output=False,
+                                                run_driver=False)
 
-        p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
-                                         compressed=False, optimizer='SLSQP', show_output=False,
-                                         run_driver=False)
-
-        html_file = pathlib.Path(_get_reports_dir(p)) / 'traj_results_report.html'
-        self.assertTrue(html_file.exists(), msg=f'{html_file} does not exist!')
+                html_file = pathlib.Path(_get_reports_dir(p)) / 'traj_results_report.html'
+                self.assertTrue(html_file.exists(), msg=f'{html_file} does not exist!')
 
     @unittest.skipIf(matplotlib is None, "This test requires matplotlib")
     def test_mpl_plots(self):
-        dm.options['plots'] = 'matplotlib'
+        with dm.options.temporary(plots='matplotlib'):
+            p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
+                                            compressed=False, optimizer='SLSQP', show_output=False,
+                                            run_driver=False)
 
-        p = two_burn_orbit_raise_problem(transcription='gauss-lobatto', transcription_order=3,
-                                         compressed=False, optimizer='SLSQP', show_output=False,
-                                         run_driver=False)
+            expected_files = ('deltav.png', 'r.png', 'accel.png',
+                            'u1.png', 'vr.png', 'pos_x.png',
+                            'vt.png', 'pos_y.png', 'theta.png')
 
-        expected_files = ('deltav.png', 'r.png', 'accel.png',
-                          'u1.png', 'vr.png', 'pos_x.png',
-                          'vt.png', 'pos_y.png', 'theta.png')
-
-        for file in expected_files:
-            plotfile = pathlib.Path(_get_reports_dir(p)).joinpath('plots') / file
-            self.assertTrue(plotfile.exists(), msg=f'{plotfile} does not exist!')
+            for file in expected_files:
+                plotfile = pathlib.Path(_get_reports_dir(p)).joinpath('plots') / file
+                self.assertTrue(plotfile.exists(), msg=f'{plotfile} does not exist!')
 
 
 if __name__ == '__main__':  # pragma: no cover

--- a/dymos/visualization/timeseries/bokeh_timeseries_report.py
+++ b/dymos/visualization/timeseries/bokeh_timeseries_report.py
@@ -455,7 +455,7 @@ def make_timeseries_report(prob, solution_record_file=None, simulation_record_fi
             for var_name in sorted(ts_units_dict.keys(), key=str.casefold):
                 fig_kwargs = {'x_range': x_range} if x_range is not None else {}
 
-                tool_tips = [(f'{x_name}', '$x'), (f'{var_name}', '$y')]
+                tool_tips = [(f'{x_name}', f'@{x_name}'), (f'{var_name}', f'@{var_name}')]
 
                 fig = figure(tools='pan,box_zoom,xwheel_zoom,hover,undo,reset,save',
                              tooltips=tool_tips,

